MetaMask/metamask-extension
# Multichain send page components

## **Description**
This PR provides the UI for the new "send" page flow.  Nothing works functionally, simply puts the UI in to place.

## **Manual testing steps**

1.  Click the action button in the bottom footer
2. Click "Send"
3. See new send page

## **Screenshots/Recordings**

<img width="374" alt="SCR-20231016-mkrc" src="https://github.com/MetaMask/metamask-extension/assets/46655/8f9f41cc-8fe0-4e7b-914e-5d5ed226240d">


## **Related issues**

Fixes:

* https://github.com/MetaMask/MetaMask-planning/issues/1441
* https://github.com/MetaMask/MetaMask-planning/issues/1442
* https://github.com/MetaMask/MetaMask-planning/issues/1443
* https://github.com/MetaMask/MetaMask-planning/issues/1444

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
